core/domain/question_services.py

```diff
@@ -352,14 +351,9 @@ def get_question_summaries_and_skill_descriptions(
         raise Exception(
```

Comment:
If we go with this approach, we should make 3 into a named constant (MAX_SKILLS_PER_QUESTION) and, instead of having a constant for 30, use MAX_SKILLS_PER_QUESTION * NUM_QUESTIONS_PER_PAGE instead.

Reply:
Done.